packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1213,4 +1215,62 @@ class TextInput {
       args,
     );
   }
+
+  /// Finishes the current autofill context, and potentially saves the user
+  /// input for future use if `shouldSave` is true.
+  ///
+  /// {@template flutter.services.autofill.autofillContext}
+  /// An autofill context is a collection of input fields that lives in the
+  /// platform's text input plugin. The platform is encouraged to save the user
+  /// input stored in the current autofill context before the context is
+  /// destroyed, when [finishAutofillContext] is called with `shouldSave` set to
+  /// true.
+  ///
+  /// Currently, there can only be at most one autofill context at any given
+  /// time. When any input field in an [AutofillGroup] requests for autofill
+  /// (which is done automatically when an autofillable [EditableText] gains
+  /// focus), the current autofill context will merge the content of that
+  /// [AutofillGroup] into itself. When there isn't an existing autofill context,
+  /// one will be created to hold the newly added input fields from the group.
```

Comment:
Question: Since we have only one `autofill context` multiple AutofillGroups can end up in the same context right?

Reply:
Yes